pysat/tests/test_utils.py

```diff
@@ -640,3 +651,47 @@ def test_lpad(self, in_pad):
         self.out_str = pysat.utils._core.fmt_output_in_cols(self.in_str,
                                                             **self.in_kwargs)
         self.test_output()
+
+
+class TestAvailableInst():
+
+    @pytest.mark.parametrize("inst_loc", [None, pysat.instruments])
+    @pytest.mark.parametrize("inst_flag, plat_flag",
+                             [(None, None), (False, False), (True, True)])
```

Comment:
coveralls shows we are missing lines 617-620 in `display_available_instruments()` call. 

Could the second parametrize statement may end up overwriting the first one. (I checked, double statement ok)

Alternately, the number of registered modules may be 0. (I think this is it)

Reply:
The coverage in meta isn't an issue with this pull. Lines already uncovered and update here simply changes a name in those lines. 

I've offered a suggestion for adding some registered modules when running the test.